src/pages/hotel/booking/booking-confirmation/rate-details/RateDetails.scss

```diff
@@ -1,54 +1,50 @@
 .side-content {
-  padding: 2px;
-  float: left;
-  margin-top: 20px;
-  height: 550px;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
```

Comment:
gaps, booking content width, etch got changed.  better to keep default styles as such for large screens.. Also make sure there is no impact in current styles for whole page. It needs to be display as such for large screens after the changes as well..

Reply:
ok. will check